README.md

```diff
@@ -72,12 +72,9 @@ ng generate component my-new-component
 ng g component my-new-component # using the alias
 
 # components support relative path generation
-# if in the directory src/app/feature/ and you run
```

Comment:
Hm did I just found a bug then? I've tried this feature numerous times but I couldn't actually do this myself. Here's the `history` of what I did on `1.0.0`.

```bash
ng new test-cli
cd test-cli
cd src
ls
cd app
mkdir feature
cd feature
ng g component new-cmp
```